### PR TITLE
Pause permissions

### PR DESCRIPTION
**Pull request by stu-gott**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1871775

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Fixed bug preventing non-admin users from pausing/unpausing VMs
```

### CODE REVIEW

**Review by enp0s3 — APPROVED** [review]
/lgtm
/retest

**Comment by xpivarc** [comment]
/hold 
Kubevirt can't be deployed.

**Comment by stu-gott** [comment]
Thanks @xpivarc turns out I hadn't run "make generate" -- the operator needs permission to deploy the new/corrected permissions...

**Comment by gabrielecerami** [comment]
@stu-gott: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: manifests/generated/rbac-operator.authorization.k8s.yaml.in [hunk 1]
```diff
@@ -505,10 +505,15 @@ rules:
...
   - virtualmachineinstances/vnc
+  verbs:
+  - get
+- apiGroups:
+  - subresources.kubevirt.io
+  resources:
   - virtualmachineinstances/pause
   - virtualmachineinstances/unpause
   verbs:
-  - get
```
Comment (davidvossel) [inline manifests/generated/rbac-operator.authorization.k8s.yaml.in:511]: leave `get` here

File: manifests/generated/operator-csv.yaml.in [hunk 2]
```diff
@@ -603,10 +603,15 @@ spec:
...
           - virtualmachineinstances/vnc
+          verbs:
+          - get
+        - apiGroups:
+          - subresources.kubevirt.io
+          resources:
           - virtualmachineinstances/pause
           - virtualmachineinstances/unpause
           verbs:
-          - get
```
Comment (davidvossel) [inline manifests/generated/operator-csv.yaml.in:609]: leave `get` here.

File: manifests/generated/operator-csv.yaml.in [hunk 3]
```diff
@@ -637,10 +642,15 @@ spec:
...
```
Comment (davidvossel) [inline manifests/generated/operator-csv.yaml.in:643]: leave `get` here

File: manifests/generated/rbac-operator.authorization.k8s.yaml.in [hunk 4]
```diff
@@ -539,10 +544,15 @@ rules:
...
```
Comment (davidvossel) [inline manifests/generated/rbac-operator.authorization.k8s.yaml.in:545]: leave `get` here

**Review by davidvossel — COMMENTED** [review]
pointed out the 4 spots causing the update failure.

**Pull request merged**